tests/translator/input/api_open_api_version_match.yaml

```diff
@@ -0,0 +1,457 @@
+Transform:
```

Comment:
Why do we use such a complex template to test? Should we use a simple and short template so that we can easily know what we're trying to test here?

Reply:
It's a template when I reproduced the issue. I just deleted some unnecessary resources in the template.